Solutions/NetClean ProActive/Analytic Rules/NetClean_Sentinel_analytic_rule.yaml

```diff
@@ -0,0 +1,42 @@
+id: 77548170-5c60-42e5-bdac-b0360d0779bb
+name: NetClean ProActive Incidents
+description: |
+  'NetClean Incident'
+severity: High
+requiredDataConnectors: []
+status: Available 
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: GreaterThan
+triggerThreshold: 0
+suppressionDuration: 5h
+tactics:
+  - Discovery
+relevantTechniques:
+  - T1083
+query: |
+  Netclean_Incidents_CL |  where version_s == 1
+entityMappings:
+  - entityType: FileHash
+    fieldMappings:
+      - identifier: Value
+        columnName: md5_s
+      - identifier: Algorithm
+        columnName: MD5
+  - entityType: DNS
+    fieldMappings:
+      - identifier: DomainName
+        columnName: domain_s
+  - entityType: Host
+    fieldMappings:
+      - identifier: HostName
+        columnName: Hostname_s
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: externalIP_s
+version: 1.0.1
```

Comment:
Since this is a very generic analytical rule which converts NetClean incidents to Sentinel Incident, there should have dynamic Alert Name, Description, Tactics, Severity and any other custom details that is relevant.

Reply:
hi am not sure how to do this, could you give me an example ?